### PR TITLE
Timeline fixes

### PR DESCRIPTION
**Pull request by brantje**

### CODE REVIEW

File: js/calendar.js [hunk 1]
```diff
@@ -116,9 +116,7 @@ Calendar={
 			var percentOfDay = curSeconds / 86400;
 			//24 * 60 * 60 = 86400, # of seconds in a day
 			var topLoc = Math.floor(parentDiv.height() * percentOfDay);
-           
-			timeline.css({'left':($(".fc-agenda-axis").width()+6)+'px','top':topLoc + 'px'});
-
+			timeline.css({'left':($('.fc-today').offset().left-250),'width': $('.fc-today').width(),'top':topLoc + 'px'});
```
Comment (PVince81) [inline js/calendar.js]: Is 250 the width of something specific ? Maybe you could read that value from the object directly.
If it's the sidebar, then use `$('.app-navigation').width()` instead (or `outerWidth()` if that works better)

**Comment by PVince81** [comment]
@georgehrke please also review

**Comment by PVince81** [comment]
Looks good :+1: 
Works in IE8, Firefox 29 and Chromium

**Comment by PVince81** [comment]
Wait, there's still a bug: try resizing the window or zooming.

You might need to listen to resize and zoom events, then call your calculation function there.

**Comment by brantje** [comment]
On it

**Comment by libasys** [comment]
i am not sure if this works, cause fullcalendar.js also uses the window resize function!

**Comment by brantje** [comment]
@libasys It works here, can you test?

**Comment by libasys** [comment]
@brantje sorry i can't test it, i have a modified version of the calendar!

**Comment by PVince81** [comment]
Thing is that the resize event can be caught/used by multiple handlers, as long as you use jQuery's `$(...).resize()`. If you use `window.onresize = handler` then it might not work and might overwrite whichever handler was registered before.

**Comment by georgehrke** [comment]
:+1:

**Comment by georgehrke** [comment]
@brantje thx for your contribution :)

**Comment by georgehrke** [comment]
stable7: https://github.com/owncloud/calendar/commit/1445d6c7a71a5b13532518bb6cffbf7153ef7efd

**Pull request merged**